### PR TITLE
Api added

### PR DESCRIPTION
**Pull request by Jia-A**

### CODE REVIEW

File: src/Context/authContext.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    children
+}) => {
+    const navigate = useNavigate();
+    const [token, setToken] = useState(localStorage.getItem("token"));
+    const [user, setUser] = useState(JSON.parse(localStorage.getItem("user")));
+
+    const loginHandler = async (user) => {
+        console.log("entered login")
+        try {
+            console.log("hey")
```
Comment (kavyakunder) [inline src/Context/authContext.js:27]: remove console log

File: src/Context/authContext.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            toast.success("Login Successful")
+        } catch (err) {
+            console.log(err)
+            toast.error("Login Failed")
+        }
+
+    }
+
+    const signupHandler = async (user) => {
+        console.log("entered signup")
```
Comment (kavyakunder) [inline src/Context/authContext.js:46]: here too..

File: src/dot-components/Login/login.jsx [hunk 3]
```diff
@@ -1,43 +1,68 @@
...
-                <label for="password" class="label-inp">Password*</label>
-                <input type="password" class="input" id="password" required />
+                <fieldset>
+                    <legend for="email" className="label-inp">Email address*</legend>
+                    <input type="text" className="input" id="email" placeholder="billiejean@gmail.com"
+                        value={loginDetails.email} onChange={(e)=>setLoginDetails({...loginDetails, email :
+                    e.target.value})} required />
+                </fieldset>
+                <fieldset>
+                    <legend for="password" className="label-inp">Password*</legend>
```
Comment (kavyakunder) [inline src/dot-components/Login/login.jsx:41]: Oh, I didn't know about this tag, this was new for me!

**Review by kavyakunder — APPROVED** [review]
Code is clean and easy to understand.
Looks good to merge!

**Review by MaruthiPothuganti — APPROVED** [review]
Functionality works like a charm with clean code. PR looks good to merge.

**Pull request merged**